ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ScanConfigActionDropdown.tsx

```diff
@@ -32,11 +33,11 @@ function ScanConfigActionDropdown({
     handleGenerateDownload,
     isScanning,
     scanConfigResponse,
+    isReportJobsEnabled,
 }: ScanConfigActionDropdownProps): ReactElement {
     const history = useHistory();
     const { isFeatureFlagEnabled } = useFeatureFlags();
     const isComplianceReportingEnabled = isFeatureFlagEnabled('ROX_COMPLIANCE_REPORTING');
-    const isReportJobsEnabled = isFeatureFlagEnabled('ROX_SCAN_SCHEDULE_REPORT_JOBS');
```

Comment:
what's the logic in pulling this up when we're already checking on `ROX_COMPLIANCE_REPORTING` in this component anyways?

Reply:
maybe i'm confused -- isn't it all hooked up to the same selector in redux where the feature flag API response is held? i feel like it might be confusing if `ROX_COMPLIANCE_REPORTING` and `ROX_SCAN_SCHEDULE_REPORT_JOBS` are being handled differently in a single component

Reply:
We can also pull that up too